keychain/src/keychain.rs

```diff
@@ -45,8 +45,8 @@ impl ExtKeychain {
 }
 
 impl Keychain for ExtKeychain {
-	fn from_seed(seed: &[u8], is_floo: bool) -> Result<ExtKeychain, Error> {
-		let mut h = BIP32GrinHasher::new(is_floo);
+	fn from_seed(seed: &[u8], is_test: bool) -> Result<ExtKeychain, Error> {
```

Comment:
I have a WIP PR somewhere that attempts to cleanup passing around `is_floo` (now `is_test`). I'll see if I can resurrect it. I think we want to explicitly pass a "chain type" around rather than a simple bool like this.

Reply:
Sure, I leave it like that for now.